### PR TITLE
P0 authentication page

### PR DESCRIPTION
**Pull request by sumitk-singh**
# Authentication page created that includes-
1. Login page
2. Sign up page

### CODE REVIEW

File: pages-css/sign-up-page-css/page-style.css [hunk 1]
```diff
@@ -0,0 +1,68 @@
+:root {
+    --color-light-gray: #f2f2f2;
+    --color-dark-orange: #ff8849;
+    --color-gray: #4d4d4d;
+    --color-black: black;
```
Comment (rutu14) [inline pages-css/sign-up-page-css/page-style.css]: Use hex code, instead of color name.

File: pages-css/login-page-css/page-layout.css [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+    font-family: 'Orienta', sans-serif;
+    font-family: 'Quicksand', sans-serif;
+}
+
+
+/* component page container */
+
+.page-layout-container {
+    display: grid;
+    gap: 0.313rem;
```
Comment (bossmonkey98) [inline pages-css/login-page-css/page-layout.css:23]: Use px for values less than 10px

File: pages-css/login-page-css/page-style.css [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+    --color-dark-orange: #ff8849;
+    --color-light-orange: #ffc591;
+    --color-gray: #4d4d4d;
+    --color-dark: black;
+    --color-light-orange: #ffeed9;
+}
+
+.section-1 {
+    padding-bottom: 1rem;
+    background: var(--color-light-orange);
```
Comment (bossmonkey98) [inline pages-css/login-page-css/page-style.css:12]: try not to use bg-color for section UX isn't that appealing

File: pages-css/sign-up-page-css/page-style.css (hunk 1 above)
Comment (sumitk-singh) [reply pages-css/sign-up-page-css/page-style.css]: sure

File: pages-css/login-page-css/page-style.css [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+    background-color: var(--color-gray);
+    border-radius: 10px;
+}
+
+.login>input {
+    height: 1.25rem;
+    padding: 5px;
+}
+
+.btn-login {
```
Comment (bossmonkey98) [inline pages-css/login-page-css/page-style.css:34]: set fixed width and also
try using different color for btn

**Review by rutu14 — APPROVED** [review]
The text on button can be a little larger font size.  

The sign up link is not connected to the signup page and forgot password can be of different color.

File: pages-css/sign-up-page-css/page-style.css (hunk 1 above)
Comment (sumitk-singh) [reply pages-css/sign-up-page-css/page-style.css]: done, converted to hex

**Comment by bossmonkey98** [comment]
Position footer at the end of the page
Use hovers for signup and forget pwd
also try to increase the contrast of forget pwd text

**Review by bossmonkey98 — APPROVED** [review]

File: pages-css/login-page-css/page-style.css (hunk 3 above)
Comment (sumitk-singh) [reply pages-css/login-page-css/page-style.css:12]: ok

File: pages-css/login-page-css/page-style.css (hunk 4 above)
Comment (sumitk-singh) [reply pages-css/login-page-css/page-style.css:34]: ok

Comment (sumitk-singh) [reply pages-css/login-page-css/page-style.css:34]: done

File: pages-css/login-page-css/page-layout.css (hunk 2 above)
Comment (sumitk-singh) [reply pages-css/login-page-css/page-layout.css:23]: ok

**Pull request merged**